docs/Code-Standards.md

```diff
@@ -2,7 +2,11 @@
 
 ## Coding Approach
```

Comment:
I would remove this subheading, as it feels like unnecessary clutter. Even though I originally put it there 😅

Reply:
lol, ok